From 67e5147d92f90fd54b6b45b6a2fd69184aa7283a Mon Sep 17 00:00:00 2001
From: afl1 <afl2001@gmail.com>
Date: Wed, 7 Aug 2019 03:51:55 +0100
Subject: [PATCH 06/43] VideoPlayer: fix wrong fps

Fix wrongly reported fps, use default 59.94 fps.

Patch: kodi-le-032-fps-fix.patch
---
 xbmc/cores/VideoPlayer/VideoPlayer.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/xbmc/cores/VideoPlayer/VideoPlayer.cpp b/xbmc/cores/VideoPlayer/VideoPlayer.cpp
index 459d2aa..d3bbd1e 100644
--- a/xbmc/cores/VideoPlayer/VideoPlayer.cpp
+++ b/xbmc/cores/VideoPlayer/VideoPlayer.cpp
@@ -3595,6 +3595,10 @@ bool CVideoPlayer::OpenStream(CCurrentStream& current, int64_t demuxerId, int iS
 
     if(m_pInputStream && m_pInputStream->IsStreamType(DVDSTREAM_TYPE_DVD))
       hint.filename = "dvd";
+    if (hint.fpsrate / hint.fpsscale > 200) {
+      hint.fpsrate = 60000;
+      hint.fpsscale = 1001;
+    }
   }
   else if(STREAM_SOURCE_MASK(source) == STREAM_SOURCE_VIDEOMUX)
   {
-- 
2.7.4

